CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.1.0
```

Comment:
In future I suggest only updating the CHANGELOG on master, e.g. once things are merged. It'll make conflicts a little easier to resolve.